doc/build.gradle

```diff
@@ -95,7 +99,8 @@ createVersionFile.dependsOn cleanDoc
 sphinx.dependsOn createVersionFile
 clean.dependsOn cleanDoc
 substitute.dependsOn sphinx
-sourcesDocJar.dependsOn substitute
+deleteDocTrees.dependsOn substitute
+sourcesDocJar.dependsOn deleteDocTrees
```

Comment:
`sourcesDocJar` does not need to depend on anything as it creates the jar from the src dir where no files are generated